packages/backend-for-frontend/src/services/toolService.ts

```diff
@@ -94,10 +94,8 @@ export function toolsServiceBuilder(clients: PagoPAInteropBeClients) {
           ? toTokenValidationEService(keyEservice, keyDescriptor)
           : undefined;
 
-      const { errors: clientSignatureErrors } = verifyClientAssertionSignature(
-        clientAssertion,
-        key
-      );
+      const { errors: clientSignatureErrors } =
```

Comment:
Unrelated: could you rename this to `clientAssertionSignatureErrors`?

Reply:
Makes sense, done!